src/Commands/Sweep/SweepSitesCommand.php

```diff
@@ -12,45 +12,49 @@ class SweepSitesCommand extends SweepCommand
      * @alias sweep
      */
     public function sweepSite() {
-      $output = $this->output();
-      $user = $this->session()->getUser();
-
-      $output->writeln('<info>Gathering sites...</>');
-      $output->writeln('<info>======================================</>');
-      $this->sites()->fetch(['team_only' => true]);
-
-      // Get remove sites owned by me.
-      $me = $user->id;
-      $this->sites->filter(function($model) use ($me) {
-        if ($model->get('owner') != $me) {
-          return true;
-        }
-      });
-
-      $sites = $this->sites->serialize();
-
-      if (empty($sites)) {
-        $this->log()->notice('No results');
-      }
-      else {
-        // List all the sites.
-        foreach($sites as $site) {
-          $output->writeln($site['name']);
-        }
-        // Proceed with removing self to the team.
-        if (!$this->confirm('Are you sure you want to remove yourself to these sites?')) {
-          return;
+      if ($this->session()->isActive()) {
```

Comment:
Terminus can do this automatically for you with an [@authorize](https://github.com/pantheon-systems/terminus/blob/3.x/src/Commands/Site/InfoCommand.php#L21) annotation. This is the recommended pattern for both brevity and consistency in operation.

Reply:
Noted on this @greg-1-anderson . Updated my PR.